src/testing/complete-redux-store-dump.js

```diff
@@ -7,10 +7,8 @@ export default {
     bakery: {
       storage: {
         _store: {
-          localStorage: {
-            'https://api.jujucharms.com/identity': 'yI6WZvcmUgMjAxOS0xMS0yMVQxODo1OToxNC43NDg4MTc0ODNaIn0seyJjaWQiOiJkZWNsYXJlZCB1c2VybmFtZSBoYXRjaCJ9LHsiY2lkIjoiaHR0cDpvcmlnaW4gaHR0cDovLzE5Mi4xNjguNjQuMTY6ODAzNiJ9XSwibG9jYXRpb24iOiJpZGVudGl0eSIsImlkZW50aWZpZXIiOiJBd29RNDhMODZrc2ZzOEpsYVR5YkNMRDI5QklnWTJJMU1UQmlaRFkyTlRsbE5qUTRORE5qTUdObU9XSXhNV013TnpjM01UTWFEZ29GYkc5bmFXNFNCV3h2WjJsdSIsInNpZ25hdHVyZSI6ImQ3Y2M4MGRjOTc5MmQwMDFhY2YwYmFkYjAyM0NTM0ZDYzMmMyYzdhMzYifV0=',
-            identity: 'yI6WZvcmUgMjAxOS0xMS0yMVQxODo1OToxNC43NDg4MTc0ODNaIn0seyJjaWQiOiJkZWNsYXJlZCB1c2VybmFtZSBoYXRjaCJ9LHsiY2lkIjoiaHR0cDpvcmlnaW4gaHR0cDovLzE5Mi4xNjguNjQuMTY6ODAzNiJ9XSwibG9jYXRpb24iOiJpZGVudGl0eSIsImlkZW50aWZpZXIiOiJBd29RNDhMODZrc2ZzOEpsYVR5YkNMRDI5QklnWTJJMU1UQmlaRFkyTlRsbE5qUTRORE5qTUdObU9XSXhNV013TnpjM01UTWFEZ29GYkc5bmFXNFNCV3h2WjJsdSIsInNpZ25hdHVyZSI6ImQ3Y2M4MGRjOTc5MmQwMDFhY2YwYmFkYjAyM0NTM0ZDYzMmMyYzdhMzYifV0='
-          }
+          'https://api.jujucharms.com/identity': 'yI6WZvcmUgMjAxOS0xMS0yMVQxODo1OToxNC43NDg4MTc0ODNaIn0seyJjaWQiOiJkZWNsYXJlZCB1c2VybmFtZSBoYXRjaCJ9LHsiY2lkIjoiaHR0cDpvcmlnaW4gaHR0cDovLzE5Mi4xNjguNjQuMTY6ODAzNiJ9XSwibG9jYXRpb24iOiJpZGVudGl0eSIsImlkZW50aWZpZXIiOiJBd29RNDhMODZrc2ZzOEpsYVR5YkNMRDI5QklnWTJJMU1UQmlaRFkyTlRsbE5qUTRORE5qTUdObU9XSXhNV013TnpjM01UTWFEZ29GYkc5bmFXNFNCV3h2WjJsdSIsInNpZ25hdHVyZSI6ImQ3Y2M4MGRjOTc5MmQwMDFhY2YwYmFkYjAyM0NTM0ZDYzMmMyYzdhMzYifV0=',
+          identity: 'yI6WZvcmUgMjAxOS0xMS0yMVQxODo1OToxNC43NDg4MTc0ODNaIn0seyJjaWQiOiJkZWNsYXJlZCB1c2VybmFtZSBoYXRjaCJ9LHsiY2lkIjoiaHR0cDpvcmlnaW4gaHR0cDovLzE5Mi4xNjguNjQuMTY6ODAzNiJ9XSwibG9jYXRpb24iOiJpZGVudGl0eSIsImlkZW50aWZpZXIiOiJBd29RNDhMODZrc2ZzOEpsYVR5YkNMRDI5QklnWTJJMU1UQmlaRFkyTlRsbE5qUTRORE5qTUdObU9XSXhNV013TnpjM01UTWFEZ29GYkc5bmFXNFNCV3h2WjJsdSIsInNpZ25hdHVyZSI6ImQ3Y2M4MGRjOTc5MmQwMDFhY2YwYmFkYjAyM0NTM0ZDYzMmMyYzdhMzYifV0='
```

Comment:
Why have two identical values?

Reply:
One does not simply ask questions about the bakery and macaroons... 😆 

But let's chat about this offline and I can give you the rundown of how this works and why they are the same thing in this mock dump.